static/README.md

```diff
@@ -0,0 +1,50 @@
+# Building your own Docker static package
+
+Static packages can be built from this directory with the following syntax
+
+```shell
+make static-${TARGETOS}-${TARGETARCH}-${TARGETVARIANT}
```

Comment:
I guess we have to deal with the GOARCH vs "dpkg arch" case, otherwise these could almost be `make PLATFORM=linux/arm64 static`, following the same convention as `--platform=linux/arm64` etc

Reply:
It's not strictly Go platform but Docker ones. I was using `make TARGETPLATFORM=linux/arm64 static` previously but I think you wanted to get rid of makefile vars?

Reply:
Using `${TARGETOS}-${TARGETARCH}-${TARGETVARIANT}` is a first step to be fully sandboxed like https://github.com/docker/docker-ce-packaging/pull/661 so we would only need to call in the makefile `docker buildx bake static`.

Reply:
Switched back to `make TARGETPLATFORM=linux/arm64 static`.